rollback1.py

```diff
@@ -8,15 +8,28 @@
 
 
 class Rollback1TestCase(unittest.TestCase):
-    
+
+    def setUp(self):
+        global url, prKey, token, waitTx, host, db, pas, login
```

Comment:
Use of global variables is always bad, why not use object local variables, self.url = self.conf["url"], and so on

Reply:
I can't change that now. Such changes are needed in all tests.